### PR TITLE
Add config for two more compute nodes in devsetup/tripleo for adoption

### PR DESCRIPTION
**Pull request by marios**
This adds config in devsetup/tripleo needed for the final two compute nodes in the data-plane-adoption multinode tripleo CI job (3ctrl/3compute).

https://issues.redhat.com/browse/OSPRH-5756

### CODE REVIEW

**Comment by marios** [comment]
Testing at https://review.rdoproject.org/r/c/testproject/+/52732/8#message-4a8038b79ac64bd04f776100be1a67615ce9fa86 with the other related patches (see https://review.rdoproject.org/r/c/rdo-jobs/+/53365 and its dependencies)

Overcloud is deployed with this config https://logserver.rdoproject.org/32/52732/8/check/periodic-data-plane-adoption-multinode-osp-17-to-extracted-crc-no-ceph/0416d01/controller/ci-framework-data/logs/ci_script_010_run_tripleo.log 

```
2024-06-05 11:13:41.452 58473 WARNING tripleoclient.utils.safe_write [-] The output file /home/zuul/overcloud-deploy/overcloud/overcloud-deployment_status.yaml will be overriden[00m
Overcloud Endpoint: http://172.21.0.216:5000
Overcloud Horizon Dashboard URL: http://172.21.0.216:80/dashboard
Overcloud rc file: /home/zuul/overcloud-deploy/overcloud/overcloudrc and /home/zuul/overcloudrc
Overcloud Deployed successfully
2024-06-05 11:13:41.953 58473 INFO tripleoclient.v1.overcloud_deploy.DeployOvercloud [-] Stopping ephemeral heat.[00m
2024-06-05 11:13:42.058 58473 INFO tripleoclient.heat_launcher [-] Pod does not exist: ephemeral-heat[00m
2024-06-05 11:13:43.680 58473 INFO osc_lib.shell [-] END return value: None[00m
~
+ popd
+ exit 0
+ rv=0
+ rm -rf -- /tmp/tmp.yQ8tOX1t43
+ exit 0
```

**Comment by jistr** [comment]
lgtm

**Review by fao89 — COMMENTED** [review]
/approve

File: devsetup/tripleo/tripleo_install.sh [hunk 1]
```diff
@@ -77,7 +88,7 @@ openstack overcloud deploy --stack overcloud \
     -e /home/zuul/hostnamemap.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/docker-ha.yaml \
     -e /home/zuul/containers-prepare-parameters.yaml -e /usr/share/openstack-tripleo-heat-templates/environments/podman.yaml \
     -e /usr/share/openstack-tripleo-heat-templates/environments/low-memory-usage.yaml \
-    -e /usr/share/openstack-tripleo-heat-templates/environments/debug.yaml --validation-warnings-fatal ${CEPH_OVERCLOUD_ARGS} \
```
Comment (marios) [inline devsetup/tripleo/tripleo_install.sh:80]: green run for both ceph/non ceph job at https://review.rdoproject.org/r/c/testproject/+/52732/14#message-f95bbae393599b1c0505fe5d629cabc84623b326

**Review by fmount — APPROVED** [review]
/lgtm

**Comment by fmount** [comment]
Thank you @marios, works for me!

**Comment by fao89** [comment]
/override rdoproject.org/github-check
failure not related to the changes

**Pull request merged**